conf/application.conf

```diff
@@ -111,7 +111,7 @@ consuming-services {
 }
 
 auditing {
-  enabled=true
+  enabled=false
```

Comment:
Should this be on?

Reply:
Auditing is enabled via app-config-base for real environments - see https://github.com/hmrc/app-config-base/blob/011eaf96e94c7a392ab8d92b6b22efa558d8a612/upscan-notify.conf#L83.  This just disables it for local development to stop spamming the log when you are not running a local Datastream.